src/com/sun/ts/tests/jaxrs/ee/rs/core/response/JAXRSClient.java

```diff
@@ -1552,6 +1552,29 @@ public void readEntityGenericTypeAnnotationThrowsIllegalStateExceptionTest()
     }
   }
 
+    /*
+   * @testName: responseCreatedRelativeURITest
+   *
+   * @assertion_ids: JAXRS:JAVADOC:121;
+   *
+   * @test_Strategy: The resource calls Response.created() to set the Location header with a
+   * relative URI. The relative URI should be converted into an absolute URI by resolving it
+   * relative to the base URI.
+   */
+  public void responseCreatedRelativeURITest()
+      throws Fault {
+    String resourceUrl = getAbsoluteUrl();
+    String expected = resourceUrl.substring(0, resourceUrl.length() - "resource".length()) + "created";
+    Response response = invokeGet("created");
+    try {
+      assertFault(expected.equals(response.getHeaderString("location")),
```

Comment:
RFC 7231 allows for using both absolute and relative URLs in the HTTP location header. I assume that in the case of the relative URL this exact matching would not work.

I assume that a test for `response.getHeaderString("location").contains("created")` would work here.